README.md

```diff
@@ -93,6 +93,7 @@ Source code heuristics:
 | download-executable | Identify when a package downloads and makes executable a remote binary |
 | exec-base64 | Identify when a package dynamically executes base64-encoded code |
 | silent-process-execution | Identify when a package silently executes an executable |
+| bidirectional-characters | Identify when a package contains contains bidirectional (bidi) characters. See more at https://trojansource.codes/ |
```

Comment:
can we explain in a few words why this matters and what this could be used maliciously for?